### ISSUE TITLE
Supression for LongClassNameCheck

### ISSUE THREAD

**Issue by h1alexbel** [l3r8yJ/oop-cop#85]
Let's add a suppression mechanism for LongClassNameCheck:
```java
@SupressWarnings("OCOP.LongClassNameCheck")
class ThisIsTooLongName {}
```

**Comment by zoeself** [comment]
@h1alexbel thank you for reporting this. I'll assign someone to take care of it soon.

**Comment by zoeself** [comment]
@l3r8yJ this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2024-01-07T17:40:01.767466``.

Estimation here is ``60 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by h1alexbel** [comment]
@l3r8yJ please assign me

**Comment by zoeself** [comment]
@l3r8yJ Don't forget to close this ticket before the deadline (2024-01-07T17:40:02). You are past the first half of the allowed period.

**Comment by 0pdd** [comment]
@h1alexbel 3 puzzles [#126](https://github.com/l3r8yJ/oop-cop/issues/126), [#127](https://github.com/l3r8yJ/oop-cop/issues/127), [#128](https://github.com/l3r8yJ/oop-cop/issues/128) are still not solved.

**Comment by zoeself** [comment]
@l3r8yJ thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).

**Comment by 0pdd** [comment]
@h1alexbel 2 puzzles [#126](https://github.com/l3r8yJ/oop-cop/issues/126), [#128](https://github.com/l3r8yJ/oop-cop/issues/128) are still not solved; solved: [#127](https://github.com/l3r8yJ/oop-cop/issues/127).

**Comment by 0pdd** [comment]
@h1alexbel the puzzle [#126](https://github.com/l3r8yJ/oop-cop/issues/126) is still not solved; solved: [#127](https://github.com/l3r8yJ/oop-cop/issues/127), [#128](https://github.com/l3r8yJ/oop-cop/issues/128).
